### PR TITLE
[ZAB-13] Traning Submitting

### PR DESCRIPTION
**Pull request by Frafire**
## [ZAB-13]

---

### Summary

- Send Traning form to VATUSA

---

### Testing

1. Run all tests.

---

### Post-Deployment Validation

- Add validation steps here...

---

[ZAB-13]: https://zabartcc.atlassian.net/browse/ZAB-13?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by cdconn00** [comment]
Fixed your PR comment, please look at the styling and try to make your PR style like that. It is easier to read that way. 

Reviewing this today.

File: src/views/instructor/training/EditSession.vue [hunk 1]
```diff
@@ -143,6 +144,8 @@ export default {
 		M.CharacterCounter.init(document.querySelectorAll('textarea'), {});
 	},
 	methods: {
+		// Set all the thing
+
```
Comment (cdconn00) [inline src/views/instructor/training/EditSession.vue:148]: I don't understand this comment.

File: src/views/instructor/training/EditSession.vue [hunk 2]
```diff
@@ -173,6 +177,26 @@ export default {
 				console.log(e);
 			}
 		},
```
Comment (cdconn00) [inline src/views/instructor/training/EditSession.vue:179]: Add a blank line like you did on line 157, or delete this one.

File: src/views/instructor/training/EditSession.vue [hunk 3]
```diff
@@ -173,6 +177,26 @@ export default {
...
+				formData.append('session_date', this.session.startTime);
+				formData.append('position', this.session.position);
+				formData.append('duration', duration);
+				formData.append('movements', this.session.movements);
+				formData.append('score', this.session.progress);
+				formData.append('notes', this.session.studentNotes);
+				formData.append('location', this.session.location);
+				formData.append('ots', this.session.ots);
+				formData.append('expDate', this.$refs.expirationDate.value);
+				await vatusaApiAuth.post(`/user/${this.session.cid}/training/record`, formData);
```
Comment (cdconn00) [inline src/views/instructor/training/EditSession.vue:193]: Add new line before this.

File: src/views/instructor/training/EditSession.vue [hunk 4]
```diff
@@ -173,6 +177,26 @@ export default {
...
+
+				this.toastSuccess('Traning Record issued');
```
Comment (cdconn00) [inline src/views/instructor/training/EditSession.vue:195]: Should be ''Training Session Submitted". Or "Training Notes Submitted"

Any preference here @JWZABARTCC ?

**Review by cdconn00 — CHANGES_REQUESTED** [review]
See requested changes.

**Comment by Frafire** [comment]
> @Frafire what branch did you create `GF-submit-traning` off of?

Master

File: src/views/instructor/training/EditSession.vue (hunk 1 above)
Comment (Frafire) [reply src/views/instructor/training/EditSession.vue:148]: my fault i forgot to change that

File: src/views/instructor/training/EditSession.vue (hunk 3 above)
Comment (Frafire) [reply src/views/instructor/training/EditSession.vue:193]: before await?

**Comment by cdconn00** [comment]
> > @Frafire what branch did you create `GF-submit-traning` off of?
> 
> Master

Okay, that's making this PR a little messy. When we do a new feature we want to merge off development and then merge back into development.

Master shouldn't need to be branched off of, except in rare cases for hotfixes.

Can you make a new branch of development, and add your changes there? Then open a new PR.

**Comment by Frafire** [comment]
> > > @Frafire what branch did you create `GF-submit-traning` off of?
> 
> > 
> 
> > Master
> 
> 
> 
> Okay, that's making this PR a little messy. When we do a new feature we want to merge off development and then merge back into development.
> 
> 
> 
> Master shouldn't need to be branched off of, except in rare cases for hotfixes.
> 
> 
> 
> Can you make a new branch of development, and add your changes there? Then open a new PR.

got it, this afternoon i create new PR thanks you Cole

**Pull request closed without merging**